PowerUp/app/src/main/res/layout/intro_screen_1.xml

```diff
@@ -0,0 +1,5 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="powerup.systers.com/IntroActivity2"
+    android:background="@drawable/intro_screen1">
```

Comment:
Please don't build screens like this. The background image is already part of the app and you should add the text programatically, not as an image, so it can, for example, be translated. Make sure to put the text in `strings.xml`